lib/rules/no-container.ts

```diff
@@ -0,0 +1,120 @@
+import { ESLintUtils, TSESTree } from '@typescript-eslint/experimental-utils';
+import { getDocsUrl } from '../utils';
+import {
+  isIdentifier,
+  isMemberExpression,
+  isObjectPattern,
+  isProperty,
+  isRenderVariableDeclarator,
+} from '../node-utils';
+
+export const RULE_NAME = 'no-container';
+
+export default ESLintUtils.RuleCreator(getDocsUrl)({
+  name: RULE_NAME,
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'Disallow the use of container methods',
+      category: 'Best Practices',
+      recommended: 'error',
+    },
+    messages: {
+      noContainer:
+        'Avoid using container to query for elements. Prefer using query methods from Testing Library, such as "getByRole()"',
```

Comment:
This message and rule doc examples only mention that it's a problem to use `container` to query elements, but rule doc description and rule implementation itself are checking all `container` methods. I guess rule doc examples and this message should be updated accordingly.

Reply:
So, you're suggesting we show a message like this: `Avoid using container methods. Prefer using the methods from Testing Library, such as "getByRole()"`?